src/main/java/com/cincialert/service/UserService.java

```diff
@@ -1,37 +1,34 @@
 package com.cincialert.service;
 
+import com.cincialert.dao.UserDAO;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.cincialert.model.User;
+import com.cincialert.dto.User;
```

Comment:
Changed the calls from the service to access the DAO first. Which is how to separate business logic from application logic. The DAO will access the database